src/Moq/Properties/Resources.resx

```diff
@@ -1,366 +1,372 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
-	<!-- 
```

Comment:
Same here, please manually change indentation back from spaces to tabs so that your PR's diff doesn't include the whole file but only what you changed/added.

I know it's annoying, I'll eventually get around to fixing this problem from occurring for good.